### DISCUSSION TITLE
Hi master awawa

### DISCUSSION THREAD

**Discussion by gnisch** [awawa-dev/HyperSerialPico#15] [General]
I've fried my apa102 strip some months ago, and left hyper hdr train for some months. But decided to go back with new hardware. So I've purchased SK6812 Cold RGBW strip, adafruit itsy bitsy, and this time did solid soldering. Latest beta 20 of hyper hdr delivered on my old RPi 3b. Pi pico flashed with hyperserial pico from releases with according firmware. All this connected to Rpi3b with usb and data to strip from GPIO 2 and Power to strip on GPIO14 also strip connected to power brick. Hyper hdr detects pico, but there fun ends it's not doing anything strip is whole wite color not reacting to any change. I've tested everything fast with ESP32 and WLED, same power brick, same pi, same strip and everything works. Some ideas what could be wrong?

**Comment by awawa-dev** [comment]
Please post detailed diagram how did you connect it. You need only one gpio for led data and the other for ground

**Comment by gnisch** [comment]
Yellow wire goes to data connector on SK2861 and brown to 5v in on led strip. Its 60 x 35 x 60 x35 config. Usb from Pico to port on Rpi3, additionaly first and last segment of led strip are powered directly from power supply. 
![20231218_174047](https://github.com/awawa-dev/HyperSerialPico/assets/60313138/c9846c52-fcd4-4145-b767-727d2bfd79f5)
![2023-12-18_174639](https://github.com/awawa-dev/HyperSerialPico/assets/60313138/cee9061b-9e68-4091-9263-6649b9facb36)

**Comment by awawa-dev** [comment]
This connection is incorrect You should not use LED 5V for Pico pinout.
![obraz](https://github.com/awawa-dev/HyperSerialPico/assets/69086569/49e74bd9-46b1-4d51-84c0-d103ba529409)

**Comment by gnisch** [comment]
I can confirm, that this connection is much better :) it works. 

Thank You 
Dziękuję
